### ISSUE TITLE
miner name

### ISSUE THREAD

**Issue by psychocrypt** [alloyproject/xmr-stak-alloy#1]
Could you please rename the miner in version.cpp to 'xmr-stak-alloy' and also the binary names in CMakeList.txt.
The problem is that you are currently highjacking our version numbers and the user do not know if he is working with a fork of xmr-stak or the original. 
This will also simplify the support process if the user provides 'xmr-stak -V' output to us.

best psychocrypt

**Comment by randygrolemund** [comment]
Agreed. I had to write additional logic in my software to tell the difference between the original and this version.
